git.tex

```diff
@@ -22,21 +21,20 @@ \subsubsection{Code Organization}\label{sec:git_repositories}
 
 \subsubsection{The LSST Workflow}\label{sec:dev_workflow}
 
+%% WOM consider removing this
```

Comment:
I'd like to keep the figure if at all possible since it's a visible confirmation that we linearize our git history.